src/drive/components/LoadMore.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import { Button, Spinner } from 'cozy-ui/react'
+import { translate } from 'cozy-ui/react/I18n'
+import styles from '../styles/table'
+
+const LoadMore = ({ onClick, isLoading, text }) => (
+  <div
+    className={classnames(
+      styles['fil-content-row'],
+      styles['fil-content-row--center']
+    )}
+  >
+    <Button theme="secondary" onClick={onClick}>
+      {isLoading ? <Spinner noMargin /> : text}
+    </Button>
+  </div>
+)
+
+LoadMore.propTypes = {
+  onClick: PropTypes.func,
+  isLoading: PropTypes.bool,
+  text: PropTypes.string.isRequired
+}
+
+LoadMore.defaultProps = {
+  onClick: null,
+  isLoading: false
+}
```

Comment:
[idem](https://github.com/cozy/cozy-drive/pull/560/commits/6081429f8066159a7260512b5e1f02573e61a758#r155546200) for the default value